test/integ/test_put_get_with_gcp_account.py

```diff
@@ -23,6 +22,13 @@
 from ..generate_test_files import generate_k_lines_of_n_files
 from ..randomize import random_string
 
+try:
+    from snowflake.connector.vendored import requests  # NOQA
+    vendored_request = True
+except ImportError:  # pragma: no cover
+    import requests
```

Comment:
Where do you think you need to handle the Import Error? I belive the code will always run with the vendored requests in. this is a test code, so not that this is a problem. but I think this try/except might be not needed?

Especially that below patching explicitly patches the 'snowflake.connector.vendored.requests'. That might lead to some problems if for any reasons this import does not work and might be hard to find problems.

Reply:
Update. I see the if's below, still not sure why would you want to need that (I do not see any case where importing snoflake.connector.vendored would fail (I have not seen any exclusion while preparing packages).

Reply:
The tox environment called `olddriver` runs the newest set of our tests with an older version of the connector to make sure that we don't break compatibility, but the old version will not have the vendored `requests` and `urllib3` in it. So maybe we don't actually need to set them to `None`, but I do this trick often for type hints.